ramls/loan.json

```diff
@@ -135,8 +135,15 @@
             "name": {
```

Comment:
Given we are making a breaking change, should this be read-only as well?

Please can you check for other properties that should be read-only and aren't marked as such.

Reply:
Sure. Following properties have been declared as read only:
* item.*
* checkoutServicePoint.*
* checkinServicePoint.*

Reply:
Thank you